### ISSUE TITLE
A11y: Windows High Contrast Mode: Add outlines to Blue & Green "cards"

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#749]
## Describe the bug
1. When in Windows High Contrast Mode (LIGHT AND DARK) you cannot see the outline of the green cards & the blue areas. Add `outline: 1px solid transparent` to these divs so that an outline shows for these areas throughout the app.

Normal
<img width="772" alt="Screenshot 2024-07-15 at 10 49 47 AM" src="https://github.com/user-attachments/assets/f0c6db29-7fa1-4551-a193-7b46188f6893">

High Contrast Mode
dark
<img width="831" alt="Screenshot 2024-07-15 at 10 49 54 AM" src="https://github.com/user-attachments/assets/55979ae6-cabf-4886-b739-3e1e0e95c08e">

light
<img width="767" alt="Screenshot 2024-07-15 at 10 53 10 AM" src="https://github.com/user-attachments/assets/58db1e8c-f196-4125-83fd-a2858085ba50">

## Acceptance Criteria
Open the page in WHCM (light & dark) or simulate it using Devtools(see link below)
- [ ] The green cards & blue sections will have an outline. Should be visible in both LIGHT & DARK forced colors mode.

## Additional context
* https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/
* How to simulate WHCM in Chrome Devtools - https://devtoolstips.org/tips/en/emulate-forced-colors/

**Comment by zheyichn** [comment]
Hi, can I be assigned this ticket pls?

**Comment by bacitracin** [comment]
@zheyichn go for it!
